lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -199,66 +160,20 @@ There is no middle number. So the median of this list will be the mean of the tw
-
-@slidebreak{InvestigateC}
-
-@QandA{
-@Q{How do the mean and median compare?}
-@A{The median (11.3) is very different from the mean (39.7)!}
-
-@Q{Here we see the median (red) and mean (blue). @image{images/num-line-pounds2.png} Which do you think better represents the data?}
-@A{The median, because over half of the data is clustered quite close to it and the rest of the data is dispersed across a huge range. Very few animals have a weight close to 39.7.} 
-
-@Q{If the median were much higher than the mean, what would we expect to be true about the distribution of the dataset?}
-@A{The dataset is skewed left or has some very low outliers.}
+@lesson-instruction{
+- Complete the Median section of @opt-printable-exercise{mean-median-modes.adoc}.
+- Compare your answers with a partner.
 }
 
 @slidebreak
 
-@lesson-point{
-The @vocab{mean} is a useful calculation when all of the points are fairly balanced on either side of the middle, but it distorts things for datasets with imbalance and extreme outliers. +
-For skewed datasets, the @vocab{median} is a better summary.}
-
-=== Synthesize
-
-Mean is generally the best measure of center, because it includes information from every single point. But it's misleading for highly-skewed datasets, so statisticians fall back to the median.
-
-@QandA{
-@Q{Why would looking at the histogram for a dataset help us to decide whether *mean* or *median* would be a better measure of center?}
-@A{Median is less sensitive to skew than mean, so seeing the shape will determine whether there's a need for median over mean.}
-@Q{When there's a strong _left_ skew, will the mean be less than or greater than the median?}
-@A{Less: the left skew pulls the mean to lower values.}
-}
-
-== Mode(s) @duration{10 minutes}
-
-=== Overview
-Students learn about the mode(s) of a dataset, how to compute them, and when it is appropriate to use them as a measure of center.
-
-@ifproglang{pyret}{
-@teacher{Note: Mode(s) are often used to describe categorical data. Since Pyret can currently only calculate mode(s) from quantitative columns, we won't be discussing that in this lesson... keep your ears peeled for news of an update next year!}
-}
+==== Mode
 
-=== Launch
 The third measure of center is called the @vocab{mode(s)} of a dataset. The @vocab{mode(s)} of a dataset are the values that appear _most often_.
 
-Median and Mean always produce one number and many datasets are what we call “unimodal”, having just one mode. But sometimes there are exceptions!
+Median and mean always produce one number and many datasets are what we call “unimodal”, having just one mode. But sometimes there are exceptions!
 
 - If two or more values are equally common, there can be more than one mode.
 - If all values are equally common, then there is no mode at all!
```

Comment:
I'd start with "if all values are equally common". It's a more understandable progression to go from no modes to one more to multiple modes.

Either that, or keep the order the same but add the word "however" to the start of line 179